--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -879,7 +879,7 @@
 string (REGEX REPLACE ";" " " 3RDPARTY_NO_LIBS "${3RDPARTY_NO_LIBS}")
 if (3RDPARTY_NO_LIBS)
   if (BUILD_SHARED_LIBS)
-    message (SEND_ERROR "Could not find libraries of used third-party products: ${3RDPARTY_NO_LIBS}" )
+    message (WARNING "Could not find libraries of used third-party products: ${3RDPARTY_NO_LIBS}" )
   else()
     message (WARNING "Could not find libraries of used third-party products: ${3RDPARTY_NO_LIBS}" )
   endif()
